projects/ion/src/lib/modal/component/modal.component.html

```diff
@@ -27,7 +27,11 @@ <h4 data-testid="modalTitle">{{ configuration.title }}</h4>
       ></ion-button>
     </header>
 
-    <div tabindex="0" class="modal-body">
+    <div
+      tabindex="0"
+      class="modal-body"
+      [style.width.px]="configuration.width || 500"
```

Comment:
```suggestion
      [style.width.px]="configuration.width || DEFAULT_WIDTH"
```
You can name this value on your component.